rules/windows/image_load/sysmon_tttracer_mod_load.yml

```diff
@@ -0,0 +1,38 @@
+action: global
+title: Time Travel Debugging Utility Usage
+id: e76c8240-d68f-4773-8880-5c6f63595aaf
+description: Detects usage of Time Travel Debugging Utility. Adversaries can execute malicious processes and dump processes, such as lsass.exe, via tttracer.exe.
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Tttracer/
+    - https://twitter.com/mattifestation/status/1196390321783025666
+    - https://twitter.com/oulusoyum/status/1191329746069655553
+author: 'Ensar Şamil, @sblmsrsn, OSCD Community' 
```

Comment:
Hello @esebese !

Thank you for your contribution (: 

could you please change "OSCD Community" to "oscd.ommunity" or "@oscd_initiative"?
the thing is that oscd is an initiative, not a community.

the only place where you can see the "community" word near to the "oscd" word is the domain name.
but the only reason I've chosen this domain is that it was cheap and kinda matching the idea — it is a community activity of course, but the global infosec community, not the "oscd community".

sorry for the inconvenience.

Reply:
You got me when you said cheap (: I will change it.